gke-e2e-cluster-create/0.1/README.md

```diff
@@ -0,0 +1,43 @@
+# GKE End to End Cluster Create
```

Comment:
Any reason the name and tags all include "e2e" and "end to end"? This seems much more broadly useful than the end to end tests, even though that's what you're using it for.

Reply:
I wasn't sure what else to call it - i thought it would be doing more e2e specific stuff, but I guess the only thing is the firewall? Maybe that's a reasonable firewall to start a cluster off with? I'm happy to just call it create GKE cluster!